pagai/services/postgres/main.py

```diff
@@ -122,28 +135,45 @@ def get_table_names(connection=None):
 
     for partition in partitions:
         partition_table_names = partition[0].split(", ")
-        all_table_names = list(
-            filter(lambda x: x[0] not in partition_table_names, all_table_names)
-        )
+        all_table_names = list(filter(lambda x: x[0] not in partition_table_names, all_table_names))
```

Comment:
If I'm not wrong, you could replace this line by:
`all_table_names = [name for name in all_table_names if name[0] not in partition_table_names]`

List comprehension FTW!

Reply:
agreed, I didn't touch that line, but the formatter did ;)